src/components/cards/TeamCard.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import CardContainer from './CardContainer';
+import CardTitle from './CardTitle';
+import CardAvatar from './CardAvatar';
+
+const TeamCard = ({
+  key,
+  ownerProfileUrl,
+  avatarUrl,
+  username,
+  location,
+  starCount,
+  role
+}) => {
+  let roleClass;
+  let roleClassCardShadow;
+
+  if (role === 'Designer') {
+    roleClass = 'role-designer';
+    roleClassCardShadow = 'role-card-shadow-designer';
+  } else if (role === 'Backend') {
+    roleClass = 'role-back-end';
+    roleClassCardShadow = 'role-card-shadow-back-end';
+  }
+
+  roleClass += ' card-user-role';
+
+  return (
+    <div className={roleClassCardShadow}>
+      <CardContainer className="card-user" key={key}>
+        <CardAvatar
+          ownerProfileUrl={ownerProfileUrl}
+          avatarUrl={avatarUrl}
+          className="card-user-avatar"
+        />
+
+        <Link to={ownerProfileUrl}>
```

Comment:
Update this and pass memberId down as props from the mock repsonse
```suggestion
        <Link to={`/members/${memberId}`}>
```